orgs-using-d.dd

```diff
@@ -122,6 +122,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG Ecratum, http://www.ecratum.com, ecratum.png,
+        Supplier Relationship Management tool for SMBs,
+        Using dlang to run some of our core apps (support app, public API), 
```

Comment:
Nitpicks:

1) On the other description we just use "D"
2) Use a dot "." at the end, otherwise it will won't display
3) It's thought to be a neutral statement, so "our" doesn't work.
You can either try something like
"Core applications (support app, public API) use D."
or use quotes if you would like to make a personal statement why you have chosen D